CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# v0.2.2
```

Comment:
Should we bump to v0.3.0 for new interfaces?

Reply:
Indeed.

Reply:
Done that. Please take a look: https://github.com/rust-vmm/vmm-sys-util/commit/63ba93b340d38570bce944a240f7b88ee77085b3.

Reply:
Also change the version number here in the changelog to `v0.3.0`